index.html

```diff
@@ -1,26 +1,76 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Code Journal</title>
   <link rel="icon" href="images/favicon.ico">
-  <link
-    rel="stylesheet"
-    crossorigin="anonymous"
+  <link rel="stylesheet" crossorigin="anonymous"
     href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.14.0/css/all.min.css"
     integrity="sha512-1PKOgIY59xJ8Co8+NE6FZ+LOAZKjy+KY8iq0G4B3CyeY6wYHN3yt9PW0XpSriVlkMXe40PTKnXrLnZ9+fkDaog==">
   <link rel="stylesheet" href="css/reset.css">
   <link rel="stylesheet" href="css/layout.css">
   <link rel="stylesheet" href="css/styles.css">
 </head>
+
 <body>
+
   <header>
+    <div class="row">
+      <div class="column-full" id="nav">
+        <h2 class="container nav-item">Code Journal</h2>
+      </div>
+    </div>
   </header>
-  <main>
-    <div data-view="entry-form"></div>
-  </main>
-  <script src="js/data.js"></script>
-  <script src="js/main.js"></script>
+
+
+  <div data-view="entry form" class="entry-form">
+    <div data-view="entry form">
```

Comment:
This can be removed, you have two elements with `data-view` attributes nested in each other.